src/metadata/matroska_handler.cc

```diff
@@ -159,8 +159,6 @@ void MatroskaHandler::parseMKV(const std::shared_ptr<CdsItem>& item, std::unique
 
         el_l0 = ebml_stream.FindNextElement(LIBMATROSKA_NAMESPACE::KaxSegment_Context, i_upper_level, ~0, true);
     } // while elementLevel0
-
-    ebml_file.close();
 }
```

Comment:
That file is created in line 141 and must be closed here

Reply:
Why would the destructor not be called?

Reply:
The destructor is called by the ebml_file object cannot be closed, because its scope is the method parseMKV